test/WebJobs.Script.Tests.Integration/TestScripts/CSharp/HttpTrigger-Scenarios/run.csx

```diff
@@ -13,6 +14,10 @@ public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, Execut
 
     switch (scenario)
     {
+        case "swa":
```

Comment:
nit: could change to "staticWebApps" or add comment for easier readability?

Reply:
Now that you bring this up, I remember it took me several exposures to "SWA" to remember it stood for "static web apps." Good call on avoiding confusion.